sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/ChatClientBuilder.java

```diff
@@ -141,6 +142,20 @@ public ChatClientBuilder configuration(Configuration configuration) {
         return this;
     }
 
+    /**
+     * Sets the {@link RetryPolicy} that is used when each request is sent.
+     * <p>
+     * The default retry policy will be used in the pipeline, if not provided.
```

Comment:
I think ther e is no 'default retry policy' right? the retryPolicy by default is null.

Reply:
The default is creating a new RetryPolicy().